src/twisted/web/test/test_client.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Tests for various parts of L{twisted.web}.
+"""
+
+from twisted.internet import defer
+from twisted.trial import unittest
+
+from twisted.web import client
+
+class DummyEndPoint(object):
```

Comment:
Please add a docstring, implement `twisted.internet.endpoints.IStreamClientEndpoint`, and add a test to `HTTPConnectionPoolTests` verifies instances provide it.